rules/coding-style/tests/Rector/ClassMethod/UnSpreadOperatorRector/Fixture/skip_vendor.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Rector\CodingStyle\Tests\Rector\ClassMethod\UnSpreadOperatorRector\Fixture;
+
+final class SkipVendor
+{
+    public function run(...$args)
+    {
+    }
+
+    public function execute(array $data)
+    {
+        $this->run(...$data);
+    }
```

Comment:
In this case, the `...` should have been removed actually. It's the same file


The test I had in mind was using the `RobotLoader->addDirectory(...$directories)`

Reply:
implemented.